workspaces/vscode-python/coder.json

```diff
@@ -0,0 +1,5 @@
+{
+  "query": {
+    "folder": "/home/coder/project"
```

Comment:
What does this do?

Reply:
This wasn't part of my version. It looks like this causes code-server to focus a different project directory on startup. However, it's probably not needed, since the default is to use the current working directory (like what I had expected the EDITOR_FOCUS_DIR choice to use). Would vote to remove.

Reply:
I removed this in a commit below.